tests/test_staging/test_pg_conversion.py

```diff
@@ -77,3 +77,74 @@ def test_sic_mapper(sic_dummy_data, sic_expected_output, sic_mapper):
     df_result = sic_to_pg_mapper(sic_dummy_data, sic_mapper, target_col="201")
 
     pd.testing.assert_frame_equal(df_result, expected_output_data)
+
+
+@pytest.fixture
+def mapper():
+    mapper_rows = [
+        [36, "N"],
+        [37, "Y"],
+        [45, "AC"],
+        [47, "AD"],
+        [49, "AD"],
+        [50, "AD"],
+        [58, "AH"],
+    ]
+    columns = ["pg_numeric", "pg_alpha"]
+
+    # Create the DataFrame
+    mapper_df = pd.DataFrame(mapper_rows, columns=columns)
+
+    # Return the DataFrame
+    return mapper_df
+
+
+def test_pg_to_pg_mapper_with_many_to_one(mapper):
+    row_data = [["0001", 47, "A"], ["0001", 49, "B"], ["0002", 50, "C"]]
+    columns = ["formtype", "201", "other_col"]
```

Comment:
suggestion: 
how about formatting like this? I think this is how we've done it elsewhere in the pipeline (and in other projects I've worked on). It maybe is a little easier to see at a glance?
![image](https://github.com/ONSdigital/research-and-development/assets/105643396/bf439ac7-bb30-4c25-88b9-2d238b715f0f)